beacon-chain/lightclient/update.go

```diff
@@ -21,39 +24,27 @@ func isEmptyWithLength(bb [][]byte, length uint64) bool {
 }
 
 type Update struct {
-	Config                     *Config
-	*ethpbv2.LightClientUpdate `json:"update,omitempty"`
-}
-
-func (u *Update) computeEpochAtSlot(slot types.Slot) types.Epoch {
-	return types.Epoch(slot / u.Config.SlotsPerEpoch)
-}
-
-func (u *Update) computeSyncCommitteePeriod(epoch types.Epoch) uint64 {
-	return uint64(epoch / u.Config.EpochsPerSyncCommitteePeriod)
-}
-
-func (u *Update) computeSyncCommitteePeriodAtSlot(slot types.Slot) uint64 {
-	return u.computeSyncCommitteePeriod(u.computeEpochAtSlot(slot))
+	config                     *Config
+	*ethpbv2.LightClientUpdate `json:"update"`
 }
 
 func (u *Update) isSyncCommiteeUpdate() bool {
-	return !isEmptyWithLength(u.GetNextSyncCommitteeBranch(), ethpbv2.NextSyncCommitteeIndex)
+	return !isEmptyWithLength(u.GetNextSyncCommitteeBranch(), helpers.NextSyncCommitteeIndex)
 }
 
 func (u *Update) isFinalityUpdate() bool {
-	return !isEmptyWithLength(u.GetNextSyncCommitteeBranch(), ethpbv2.FinalizedRootIndex)
+	return !isEmptyWithLength(u.GetFinalityBranch(), helpers.NextSyncCommitteeIndex)
```

Comment:
Not sure if `helpers.NextSyncCommitteeIndex` is correct?

Reply:
good catch, fixed